### PR TITLE
Snackbar component added here

### PR DESCRIPTION
**Pull request by yogesh-**
Snackbar component added here.

image

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -1230,13 +1233,68 @@ <h2 class="h2-style">Grid - Three Column</h2>
...
+                <div id="snackbar">
+
+                    <h1 class="h1-style">Snackbar</h1>
+                    <p class="p-style">Snackbars inform users of a process that an app/website has
+                        performed or will perform.
+                        They appear temporarily, towards the bottom of the screen.
+                        They shouldn’t interrupt the user experience, and they generally don’t require
+                        user input to disappear.
+
+                    </p>
```
Comment (tushargoyal812) [inline documentation.html]: remove unnecessary space from here

File: documentation.html [hunk 2]
```diff
@@ -1230,13 +1233,68 @@ <h2 class="h2-style">Grid - Three Column</h2>
...
+
+                    <p class="p-style">Simple snackbar consits of elements stacked in a single row
+                        with
+                        action buttons generally on the right.<br>
+                        Use class of <span>snackbar</span> on the main class with elements inside it.
+
+
```
Comment (tushargoyal812) [inline documentation.html]: also here

File: documentation.html [hunk 3]
```diff
@@ -1230,13 +1233,68 @@ <h2 class="h2-style">Grid - Three Column</h2>
...
+
+    </div>
+    </div>
+
+
+
+
+
```
Comment (tushargoyal812) [inline documentation.html]: and here also i will good for indentation

File: css/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  flex-direction: row;
+  justify-content: space-around;
+  margin-bottom: 5rem;
+  width: 40rem;
+  background: var(--charcoal-black);
+  color: var(--font-color-special);
+  padding: 2rem 0rem;
+  font-size: 1.5rem;
+  margin: 1rem auto;
+}
```
Comment (shivanigangadharan) [inline css/snackbar.css:12]: Suggestion: can add a hover effect on 'RETRY' and 'X' like `cursor: pointer`, since they will become clickable later on.

**Review by shivanigangadharan — APPROVED** [review]
Looks good to merge. 👍

**Review by tushargoyal812 — COMMENTED** [review]
i think everything looks good just take care of indentation only

File: css/snackbar.css (hunk 4 above)
Comment (yogesh-) [reply css/snackbar.css:12]: ok

Comment (yogesh-) [reply css/snackbar.css:12]: done

File: documentation.html (hunk 1 above)
Comment (yogesh-) [reply documentation.html]: ok

File: documentation.html (hunk 3 above)
Comment (yogesh-) [reply documentation.html]: ok

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

**Pull request merged**